source/extensions/filters/http/common/compressor/compressor.cc

```diff
@@ -109,6 +109,10 @@ CompressorFilterConfig::ResponseDirectionConfig::ResponseDirectionConfig(
           proto_config.has_response_direction_config()
               ? proto_config.response_direction_config().remove_accept_encoding_header()
               : proto_config.remove_accept_encoding_header()),
+      always_compress_content_if_exists_(
+          proto_config.has_response_direction_config()
```

Comment:
This can be abbreviated to just `proto_config.response_direction_config().always_compress_content_if_exists()`, since proto has matching default semantics.